content/en/platform/corda/4.12/enterprise/node/operating/tvu/testing-tvu.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
@suhas-srivastava  This page needs re-doing as customers don't have access to the samples directory. They don't have access to enterprise source.

The code and build.gradle file required here is so small, that it could actually be in the doc. @suhas-srivastava can you prepare some doc explaining what to do, and inserting the code required in. But build it locally first.